docs/reference/api/docker_remote_api.md

```diff
@@ -117,6 +117,7 @@ This section lists each version from latest to oldest.  Each listing includes a
 [Docker Remote API v1.25](docker_remote_api_v1.25.md) documentation
 
 * `POST /containers/create` now takes `AutoRemove` in HostConfig, to enable auto-removal of the container on daemon side when the container's process exits.
+* `GET /containers/json` now adds `removing` to the container state.
```

Comment:
I think there's a two changes, first of all, it *shows* "removing" as a state for containers (this is also true for the "inspect a container" endpoint). The second is that `/containers/json` now allows *filtering* on that state. Something like this perhaps? (better suggestions welcome);


```Markdown

* `GET /containers/json` and `GET /containers/(id or name)/json` now return `"removing"` 
  as a value for the `. State.Status` field if the container is being removed. Previously,
  "exited" was returned as status.
* `GET /containers/json` now adds `removing` as a valid value for the `status` filter.
```